doc/README.hi.md

```diff
@@ -0,0 +1,143 @@
+जितने बदलाव हो यह उतना ही बेहतर होता जाता है!
+
+अगर आप शामिल होना चाहते हैं, हमारे [योगदान की टिप्पणियाँ](.github/CONTRIBUTING.md) देखें।
+
+<a href="https://hosted.weblate.org/engage/newpipe/">
+<img src="https://hosted.weblate.org/widgets/newpipe/-/287x66-grey.png" alt="अनुवाद की स्थिति" />
+</a>
+
+## दान करें
+अगर आपको NewPipe पसंद है, एक छोटे-से दान से हम खुश होंगे। आप या तो बिटकॉइन भेज सकते हैं या फिर Bountysource या Liberapay से दान कर सकते हैं। NewPipe को दान करने के बारे में अधिक जानकारी के लिए कृपया हमारा [वेबसाइट](https://newpipe.net/donate) देखें।
+
+<table>
+  <tr>
+    <td><img src="https://bitcoin.org/img/icons/logotop.svg" alt="Bitcoin"></td>
+    <td><img src="../assets/bitcoin_qr_code.png" alt="बिटकॉइन पर दान करने के लिए QR कोड" width="100px"></td>
+    <td><samp>16A9J59ahMRqkLSZjhYj33n9j3fMztFxnh</samp></td>
+  </tr>
+  <tr>
+    <td><a href="https://liberapay.com/TeamNewPipe/"><img src="https://upload.wikimedia.org/wikipedia/commons/2/27/Liberapay_logo_v2_white-on-yellow.svg" alt="Liberapay" width="80px"></a></td>
+    <td><a href="https://liberapay.com/TeamNewPipe/"><img src="../assets/liberapay_qr_code.png" alt="liberapay.com पर NewPipe को देखें" width="100px"></a></td>
+    <td><a href="https://liberapay.com/TeamNewPipe/donate"><img src="../assets/liberapay_donate_button.svg" alt="Liberapay के ज़रिए दान करें" height="35px"></a></td>
+  </tr>
+  <tr>
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/22/Bountysource.png/320px-Bountysource.png" alt="Bountysource" width="190px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe"><img src="../assets/bountysource_qr_code.png" alt="bountysource.com पर NewPipe को देखें" width="100px"></a></td>
+    <td><a href="https://www.bountysource.com/teams/newpipe/issues"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f" height="30px" alt="देखें कि आप कितने बाउंटी कमा सकते हैं।"></a></td>
+  </tr>
+</table>
+
+## निजता नीति
+
+NewPipe परियोजना का लक्ष्य है मीडिया वेब सेवाओं का इस्तेमाल करते हुए एक निजी, गुमनाम अनुभव प्रदान करना,
+इसलिए यह ऐप आपकी अनुमति के बिना कोई भी डेटा प्राप्त नहीं करती है। NewPipe की निजता नीति में विस्तार से बताया गया है कि क्रैश रिपोर्ट भेजते समय या हमारे ब्लॉग पर टिप्पणी जोड़ते समय कौन-कौन-सी डेटा भेजी जाती है। आपको दरख़्वास्त [यहाँ](https://newpipe.net/legal/privacy/) मिल जाएगा।
+
+## लाइसेंस
+[![GNU GPLv3 का चित्र](https://www.gnu.org/graphics/gplv3-127x51.png)](https://www.gnu.org/licenses/gpl-3.0.en.html)  
+
+NewPipe मुक्त सॉफ़्टवेयर है: आप इच्छानुसार इसे सुधार सकते हैं, जाँच सकते हैं, बाँट सकते हैं और इस्तेमाल कर सकते हैं। विशिष्ट रूप से, आप मुक्त सॉफ़्टवेयर संस्थान द्वारा परिभाषित [GNU साधारण सार्वजनिक लाइसेंस](https://www.gnu.org/licenses/gpl.html) के संस्करण 3 या फिर वैकल्पिक रूप से किसी नवीन संस्करण के शर्तों के अनुसार इसे दोबारा बाँट सकते हैं और/या बदल सकते हैं।

```

Comment:
Is "mukt software" the generally accepted translation of "free software"? I would say use "open source" here as well, like at the beginning of the Readme.